tests/snapshots/tests.contrib.starlette.test_starlette.test_schematization[service_schema4]_rest.json

```diff
@@ -2,7 +2,7 @@
   {
     "name": "starlette.request",
     "service": "mysvc",
-    "resource": "GET /sub-app/hello/name",
+    "resource": "GET /sub-app/hello/{name}",
```

Comment:
I wouldn't expect the removal of deprecated stuff to cause any external behavior (like the emitted spans captured in snapshots) to change. How's this related?

Reply:
This is because the test in question `test_schematization` explicitly set `aggregate_resources=False`, but that is no longer an allowed behavior (we will always aggregate resources by default), meaning the asserted resource name should include the `{}` encapsulating the route name. This PR should include some more context: https://github.com/DataDog/dd-trace-py/pull/3612